src_py/locals.py

```diff
@@ -83,6 +83,13 @@
     "BUTTON_WHEELUP",
     "BUTTON_X1",
     "BUTTON_X2",
+    "APP_TERMINATING",
+    "APP_LOWMEMORY",
+    "APP_WILLENTERBACKGROUND",
+    "APP_DIDENTERBACKGROUND",
+    "APP_WILLENTERFOREGROUND",
+    "APP_DIDENTERFOREGROUND",
+    "CLIPBOARDUPDATE",
```

Comment:
It looks like these are mostly alphabetical, so that should be preserved I guess?

Reply:
true